two_factor/forms.py

```diff
@@ -176,3 +177,31 @@ def clean(self):
 
 class BackupTokenForm(AuthenticationTokenForm):
     otp_token = forms.CharField(label=_("Token"))
+
+
+class TokenAuthenticationForm(AuthenticationForm):
```

Comment:
Because I wanted to keep using the normal login form behaviour (especially the `confirm_login_allowed` step + I wanted to make it transparant for the view whether the user is properly authenticated or it is a fake login